tests/analyzer/sourcecode/code-execution.py

```diff
@@ -98,16 +98,13 @@ def get_version():
 
 def RunCustomCommand(command_list):
     print("Running command: %s" % command_list)
-    # ok: code-execution
+    # ruleid: code-execution
```

Comment:
note: I'm unsure why this code was deemed to be acceptable, I believe it should generate a match